### PR TITLE
Feature/twitter link

### PR DESCRIPTION
**Pull request by sethlaw**
Hey Chris, just want you to check over my logic on this. I've tested and pushed it to Testflight as well. Will be version 3.0.4.

### CODE REVIEW

File: hackertracker/HTEventDetailViewController.swift [hunk 1]
```diff
@@ -200,15 +206,23 @@ class HTEventDetailViewController: UIViewController {
                 if speakers.count > 1, i < speakers.count {
                     speakerBios.append(NSAttributedString(string:"\n\n"))
                 }
+                
+                if let twitter = s.twitter {
```
Comment (chrsmys) [inline hackertracker/HTEventDetailViewController.swift]: You can combine these two if statements

File: hackertracker/HTEventDetailViewController.swift [hunk 2]
```diff
@@ -200,15 +206,23 @@ class HTEventDetailViewController: UIViewController {
...
+                    if twitter != "" {
+                        NSLog("adding twitter button for \(twitter)")
```
Comment (chrsmys) [inline hackertracker/HTEventDetailViewController.swift]: We should remove logging

File: hackertracker/HTMyScheduleTableViewController.swift [hunk 3]
```diff
@@ -22,14 +22,18 @@ class HTMyScheduleTableViewController: BaseScheduleTableViewController {
...
+        let fr = NSFetchRequest<NSFetchRequestResult>(entityName:"Event")
         let startofDay: Date = DateFormatterUtility.yearMonthDayTimeFormatter.date(from: "\(dateString) 00:00:00 PDT")!
         let endofDay: Date = DateFormatterUtility.yearMonthDayTimeFormatter.date(from: "\(dateString) 23:59:59 PDT")!
-
-        let fr = NSFetchRequest<NSFetchRequestResult>(entityName:"Event")
-        fr.predicate = NSPredicate(format: "start_date >= %@ AND end_date <= %@ AND starred == YES", argumentArray: [startofDay, endofDay])
+        
+        if let con = DataRequestManager(managedContext: getContext()).getSelectedConference() {
+            fr.predicate = NSPredicate(format: "start_date >= %@ AND start_date <= %@ AND starred == YES AND conference = %@", argumentArray: [startofDay, endofDay,con])
+
```
Comment (chrsmys) [inline hackertracker/HTMyScheduleTableViewController.swift]: Remove empty line

File: hackertracker/UpdateCell.swift [hunk 4]
```diff
@@ -36,14 +36,19 @@ public class UpdateCell : UITableViewCell {
     }
     
     func bind(vendor: Vendor) {
-        let date = DateFormatterUtility.yearMonthDayNoTimeZoneTimeFormatter.string(from: vendor.updated_at!)
+        //let date = DateFormatterUtility.yearMonthDayNoTimeZoneTimeFormatter.string(from: vendor.updated_at!)
```
Comment (chrsmys) [inline hackertracker/UpdateCell.swift]: Remove

**Comment by sethlaw** [comment]
yeah, already merged

**Pull request closed without merging**